README.md

```diff
@@ -5,6 +5,21 @@
 Sample [Autodesk Forge](https://forge.autodesk.com) application attempting to provide a cleaner,
 easier-to-read implementation of the "View Models" application from https://learnforge.autodesk.io.
 
+## Setup & Run
+
+- Clone this repository
+- Install dependencies: `dotnet restore`
+- Setup environment variables in the appsettings.Development.json (create this file if its missing):
+  - `FORGE_CLIENT_ID` - your Forge application client ID
+  - `FORGE_CLIENT_SECRET` - your Forge application client secret
+  - `FORGE_BUCKET` - name of Forge bucket to store your designs in
+
+```
+Remember to replace the client ID, secret and forge bucket with your own. Note that the bucket name should be unique to your application.
+Using a bucket name that is already in use with different forge app credentials (client Id and Secret) to upload a file will result in an error.
```

Comment:
The unique name requirement is not related to file uploads. The requirement is enforced as soon as the developer attempts to create the bucket. I would suggest something like "Creating a bucket with a name that is already used/owned by other Forge application will result in a 409 error".